src/js/firstparties/facebook.js

```diff
@@ -1,6 +1,6 @@
 // Adapted from https://github.com/mgziminsky/FacebookTrackingRemoval
 (function() {
-let fb_wrapped_link = "a[href*='facebook.com/l.php?'";
+let fb_wrapped_link = `a[href*='${document.domain}/l.php?'`;
```

Comment:
I'm concerned about trusting `document.domain` ... Could this be abused in some way?

Should we instead have a selector that says the link's href property points to `facebook.com/l.php?` or `facebookcorewwwi.onion/l.php?`? I think we can do that as `SELECTOR, SELECTOR`.

Reply:
Looks good otherwise.

Reply:
For posterity: since this script runs in its own context, I don't think we need to worry about `document.domain` being spoofed by outside JS.